README.md

```diff
@@ -1384,6 +1386,56 @@ out  | evt.ota_end.report      | object     | Sent on upgrade end with upgrade s
    }
 }
 ```
+### Virtual meter service
+
+This service enables a device to report accumulated consumption in case it does not have its own metering capabilities (eg. a relay with output binary switch service or thermostat). Thanks to the service accumulated consumption is calculated based on manually provided consumption in a unit of time (eg. watts, m3/h). A device shall calculate cumulated consumption at every state's change, mode change and at least every interval value - 30 minutes by default. When this service is added on a device, appropriate metering service is created (eg. meter_elec, meter_water) and a device starts to send measurements.
+
+#### Service names
+
+`meter_virtual`
+
+#### Interfaces
+
+Type | Interface                | Value type |   Unit  | Description
+-----|--------------------------|------------|---------|------------
+in   | cmd.set_interval         | int        | minutes | Interval  for accumulated consumption recalculation. Overwrites a default value.
+in   | cmd.add                  | int_float  | define by `unit` props   | Adds Virtual meter service to a selected device to report accumulated consumption. Map of floats shall provide consumption for every mode.
+in   | cmd.remove               | null       |         | Removes Virtual meter service from a selected device. The device shall not be reporting accumulated consumption.
+
+  #### Interface props
+
+Name         | Value example | Description
+-------------|---------------|-------------
+`unit`       | "W", "m3/h"   |  Consumption
+  
+#### Examples
+
+Adding Virtual meter service on a device working in a one of three available modes "off", "heat" or "fan":
+
+```json
+{
+   "type": "cmd.add_device",
+   "serv": "meter_virtual",
+   "val_t": "int_float",
+   "val": {
+      "off": 10,
+      "heat": 1500,
+      "fan": 250
+   }
+}
+```
+
+Virtual meter service measurement reporting energy delivered value equal 123.5 kWh:
```

Comment:
Not sure if this example is important, because it will be reported BY the meter elec service not virtual meter service.

Reply:
This is just an exemple, should be just helpful.

Reply:
I can move it to meter_elec paragraph and put a link here, what do you thinkg is better ?